server-kotlin/Dockerfile

```diff
@@ -4,10 +4,8 @@ COPY . /home/gradle/server
 WORKDIR /home/gradle/server
 RUN gradle build -x test
```

Comment:
Pro zjednoduseni bych zvazil moznost pouzit plugin application:

`
plugins {
    id 'org.springframework.boot' version "2.2.5.RELEASE"
    id 'io.spring.dependency-management' version "1.0.9.RELEASE"
    id 'org.jetbrains.kotlin.jvm' version "$kotlin_version"
    id 'org.jetbrains.kotlin.plugin.spring' version "$kotlin_version"
    id 'com.google.cloud.tools.jib' version '2.1.0'
    id 'application'
}

application {
    mainClassName = 'digital.cesko.Application'
}
`

A potom buildit s `RUN gradle installBootDist --no-daemon`, pripadne custom install s testy, odstrani se tim zavislost na verzi v Dockerfile.

Reply:
Dik, to jsem neznal, zkusim

Reply:
Jestli neva, tak bych to ted mergnul a pak to vylepsil. Nevim kdy na to budu mit cas a nechci nechat ten docker-compose rozbity zbytecne dlouho.